### DISCUSSION TITLE
Prevent update dotnet monorepo (major)

### DISCUSSION THREAD

**Discussion by jokay** [renovatebot/renovate#9730] [Request Help] [answered]
We have a repo which uses **.NET Core 3.1** (LTS). Renovate proposes to update to **.NET Core 5.0**.
We want to use the **LTS** version and this is an **Immortal** merge request :wink:

This is the `renovate.json` we use:

```json
{
    "extends": [
        "config:base",
        "docker:enableMajor"
    ]
}
```

Is there a way to prevent this merge request by using `packageRules` without having to add every single NuGet package?
Or is there a better way of handling this kind of use case?

![monorepo-dotnet](https://user-images.githubusercontent.com/18613935/116109191-db63b180-a6b4-11eb-9d26-72c7f3e8a1fa.png)

**Comment by rarkins** [comment, marked as answer]
I think this should work:

```json
{
  "packageRules": [{
    "extends": ["monorepo:dotnet"],
    "allowedVersions": "<4"
  }]
}
```

**Reply by jokay** [reply]
Thank you very much, this worked :+1:

```sh
INFO: Autoclosing PR (repository=test/demo)
       "branchName": "renovate/major-dotnet-monorepo",
       "prNo": 77,
       "prTitle": "Update dotnet monorepo (major)"
```
